src/microceph.py

```diff
@@ -333,6 +333,33 @@ def set_pool_size(pools: str, size: int):
     _run_cmd(cmd)
 
 
+def exit_maintenance(name: str, dry_run: bool = False) -> str:
+    """Bring the ceph node out of maintenance mode."""
+    cmd = ["sudo", "microceph", "cluster", "maintenance", "exit", name]
+    cmd.extend([f"--dry-run={dry_run}"])
+    return _run_cmd(cmd)
+
+
+def enter_maintenance(
+    name: str,
+    set_noout: bool = True,
+    stop_osds: bool = False,
+    force: bool = False,
+    dry_run: bool = False,
```

Comment:
These arguments do not need to be optional arguments - the callers always provide all arguments. Please only use optional arguments if it's really needed, as they can hide errors like accidentally omitting an argument.

```suggestion
    set_noout: bool,
    stop_osds: bool,
    force: bool,
    dry_run: bool,
```

Reply:
Fixed